docs/data/date-pickers/localization/data.json

```diff
@@ -142,5 +142,13 @@
     "missingKeysCount": 7,
     "totalKeysCount": 35,
     "githubLink": "https://github.com/mui/mui-x/blob/next/packages/x-date-pickers/src/locales/urPK.ts/"
+  },
+  {
+    "languageTag": "be-BY",
```

Comment:
Could you also apply the following diff and run `yarn l10n` to generate data for translations in correct alphabetical order. 🙏 

```diff
diff --git a/scripts/localeNames.js b/scripts/localeNames.js
index d7790b9b7..a4ccef65b 100644
--- a/scripts/localeNames.js
+++ b/scripts/localeNames.js
@@ -6,6 +6,7 @@ module.exports = {
   'hy-AM': 'Armenian',
   'az-AZ': 'Azerbaijani',
   'bn-BD': 'Bangla',
+  'be-BY': 'Belarusian',
   'bg-BG': 'Bulgarian',
   'ca-ES': 'Catalan',
   'zh-HK': 'Chinese (Hong Kong)',
```
 
I've tried pushing to your remote, but it was probably denied, because you are using `next` branch.

Reply:
done :slightly_smiling_face:

Reply:
Thank you once again! 👍

Reply:
You are welcome :blush: